src/ipcache.cc

```diff
@@ -763,12 +763,12 @@ ipcacheStatPrint(ipcache_entry * i, StoreEntry * sentry)
     char buf[MAX_IPSTRLEN];
 
     if (!sentry) {
-        debugs(14, DBG_CRITICAL, HERE << "CRITICAL: sentry is NULL!");
+        debugs(14, DBG_CRITICAL, "CRITICAL: sentry is NULL!");
```

Comment:
Agreed, but this PR automatically removes HERE, and nothing else should be required.